src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/BsnesApi.cs

```diff
@@ -116,9 +116,9 @@ public BsnesApi(string dllPath, CoreComm comm, IEnumerable<Delegate> allCallback
 				Filename = "bsnes.wbx",
 				Path = dllPath,
 				SbrkHeapSizeKB = 14 * 1024,
-				InvisibleHeapSizeKB = 4,
+				InvisibleHeapSizeKB = 200 * 1024,
 				MmapHeapSizeKB = 110 * 1024, // TODO: check whether this needs to be larger; it depends on the rom size
-				PlainHeapSizeKB = 0,
+				PlainHeapSizeKB = 110 * 1024,
```

Comment:
These should ideally be adjusted to the lowest size that consistently works (and you can reduce the `MmapHeapSize` and `SbrkHeapSize` in the process`)